capstone/src/main/webapp/dashboard.html

```diff
@@ -6,14 +6,16 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getClassrooms()">
 
     <div id="content">
```

Comment:
The IDs here are a little confusing. This implies this is the content of the page, but it looks like its... The creation modal?

Reply:
removed unnecessary div